libraries/common/effekt.effekt

```diff
@@ -649,6 +665,12 @@ extern pure def bitwiseXor(x: Int, y: Int): Int =
   llvm "%z = xor %Int ${x}, ${y} ret %Int %z"
   vm "effekt::bitwiseXor(Int, Int)"
 
+extern pure def bitwiseNot(x: Int): Int =
+  js "(~${x})"
+  chez "(lognot ${x})"
+  llvm "%z = xor %Int ${x}, -1 ret %Int %z"
+  vm "effekt::bitwiseNot(Int)"
```

Comment:
Is this actually implemented in the VM? 🤔

Reply:
I don't think so, actually. I'm not sure how we handled this in other cases (I don't think all of them are implemented, are they?)

Reply:
... they are, I think? https://github.com/effekt-lang/effekt/blob/f175258a6800df6a760e7323ea9333a769e369d1/effekt/shared/src/main/scala/effekt/core/vm/Builtin.scala#L121-L152

Reply:
Thx, it is now implemented.